### ISSUE TITLE
Typescript Action Type Definition

### ISSUE THREAD

**Issue by hadigoh** [rt2zz/redux-persist#931]
Shouldn't [this line](https://github.com/rt2zz/redux-persist/blob/208c6b112ead87b3701dfacaae2cdbe78377775a/src/index.d.ts#L141) instead of:
```
type: "redux-persist/es/REHYDRATE",
```

should be:
```
type: "persist/REHYDRATE",
```

Matching the value from [here](https://github.com/rt2zz/redux-persist/blob/208c6b112ead87b3701dfacaae2cdbe78377775a/src/index.d.ts#L99)?
```
export const REHYDRATE: "persist/REHYDRATE";
```

This prevents switch case by `action.type` like below:
```
function reducer<T>(state: T, action: PersistorAction): T {
  switch (action.type) {
  case REHYDRATE: // this fails as 'redux-persist/es/REHYDRATE' != 'persist/REHYDRATE'
    // do something
    break;
  }
}
```

**Comment by aguynamedben** [comment]
I think this might be fixed in a recently merged PR: https://github.com/rt2zz/redux-persist/pull/919/files#diff-13437120c94446e7f7325a148bcc9694R105

To try the new version out follow the instructions in this comment: https://github.com/rt2zz/redux-persist/pull/919#issuecomment-445576500

Can you try out 5.11.0-alpha and let me know if that solves the issue you found @hadigoh?

**Comment by aguynamedben** [comment]
@hadigoh I haven't heard back from you. Let me know if you find this is not fixed with the PR I referenced that's included in 5.11.0-alpha.

**Comment by hadigoh** [comment]
@aguynamedben Yes that should fix this issue :+1:
